documentation/troubleshooting.md

```diff
@@ -184,3 +184,70 @@ certificate and certificate directory.
 Visit the **#bolt** channel in the [Puppet Community
 Slack](https://slack.puppet.com) to find a whole community of people waiting
 to help!
+
+## PowerShell does not recognize Bolt cmdlets
+
+PowerShell 3.0 cannot automatically discover and load the Bolt module. If you're
+using PowerShell 3.0, add the Bolt module manually.
+
+> 🔩 **Tip** To confirm your PowerShell version, run`$PSVersionTable`.
+
+To allow PowerShell to load Bolt, add the correct module to your PowerShell
+profile.
+
+1.  Update your PowerShell profile.
+    ```
+    'Import-Module -Name ${Env:ProgramFiles}\WindowsPowerShell\Modules\PuppetBolt' | Out-File -Append $PROFILE
+    ```
+1.  Load the module in your current PowerShell window.
+    ```
+    . $PROFILE
+    ```
+
+## PowerShell could not load the Bolt PowerShell module
+
+PowerShell's execution policy is a safety feature that controls the conditions
+under which PowerShell loads configuration files and runs scripts. This feature
+helps prevent the execution of malicious scripts. The default policy is
+`Restricted` (allow no scripts to run) for Windows _clients_ and `RemoteSigned`
```

Comment:
I'm not sure what Clients means here? Desktops? Or just non-Windows Server?

Reply:
This is from their verbiage: https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-7.1

I think "client" means a machine not running the Windows Server OS. We link to the page below, so maybe that's enough for users. I agree it's not super clear, but we're using their language....

Reply:
Today I learned! Yeah, using their specified language seems like the right call.